tsconfig-base.json

```diff
@@ -8,6 +8,11 @@
         "noUnusedParameters": true,
         "noImplicitReturns": true,
         "noFallthroughCasesInSwitch": true,
-        "composite": true
+        "composite": true,
+		"baseUrl": "./",
+		"paths": {

```

Comment:
If we had a sufficiently intelligent linting rule, this would be the only place that "packages" get defined.